hapistrano.cabal

```diff
@@ -1,5 +1,5 @@
 name:                hapistrano
-version:             0.3.8.0
+version:             0.3.9.0
```

Comment:
I think we can merge both PRs under the same version

Reply:
Sounds good to me.